assets/src/edit-story/components/canvas/karma/rightClickMenu.karma.js

```diff
@@ -18,31 +18,121 @@
  */
```

Comment:
A lot of this is copied form #8382. There are only two new tests, which I'll mark. These are smoke tests since the unit tests cover the actual changed data.